pkg/executor/aggregate/agg_hash_partial_worker.go

```diff
@@ -108,6 +106,17 @@ func (w *HashAggPartialWorker) fetchChunkAndProcess(ctx sessionctx.Context, hasE
 
 	defer w.inflightChunkSync.Done()
```

Comment:
I think in theory, you should register the defer function right after L100

Reply:
> I think in theory, you should register the defer function right after L100

done